pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -184,4 +184,13 @@ under the License.
             <resolver>org.apache.hawq.pxf.plugins.jdbc.JdbcReadResolver</resolver>
         </plugins>
     </profile>
+    <profile>
+        <name>Parquet</name>
```

Comment:
if someone could create another connector/profile that would access parquet files from a different external data store, we should consider qualifying the profile name, i.e. HdfsParquet.  (along the same lines, perhaps we should qualify the profiles named Avro and Json.)

Reply:
The storage layer is determined in fragmenter and accessor, resolver should remain the same. I would leave it as Parquet and later reconsider storage vs format naming conventions for all profiles.